sklearn/neural_network/_rbm.py

```diff
@@ -57,9 +57,12 @@ class BernoulliRBM(TransformerMixin, BaseEstimator):
         The verbosity level. The default, zero, means silent mode.
 
     random_state : integer or RandomState, default=None
-        A random number generator instance to define the state of the
-        random permutations generator. If an integer is given, it fixes the
-        seed. Defaults to the global numpy random number generator.
+        Determines random number generation for Gibbs sampling from visible
+        and hidden layers; for initializing components, sampling from
+        visible layers and sampling hidden activations during fit and partial
```

Comment:
I don't think we need to specify in which method. We don't do it in other places.